paymentapi/src/main/java/fi/hel/verkkokauppa/payment/service/PaymentFilterService.java

```diff
@@ -0,0 +1,49 @@
+package fi.hel.verkkokauppa.payment.service;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import fi.hel.verkkokauppa.payment.api.data.PaymentFilterDto;
+import fi.hel.verkkokauppa.payment.model.PaymentFilter;
+import fi.hel.verkkokauppa.payment.repository.PaymentFilterRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class PaymentFilterService {
+    @Autowired
+    private ObjectMapper objectMapper;
+    @Autowired
+    private PaymentFilterRepository paymentFilterRepository;
+
+    public List<PaymentFilter> savePaymentFilters(List<PaymentFilterDto> paymentFiltersDto) {
```

Comment:
This function allow multiple filters to be created with same id, this should be fixed